OpenMTRDemo/Filters/BaseFilter.cs

```diff
@@ -82,17 +83,5 @@ private void moveButton_Click(object sender, EventArgs e)
             catch (NullReferenceException) { }
             catch (IndexOutOfRangeException) { }
```

Comment:
Think we should give the user some information regarding these exceptions?

Reply:
For the null reference sure, I can add a message box. But the index out of range happens if the filter is on the bottom or top and you try to move it down or up respectively. It makes more sense to me for nothing to happen there.